users/models.py

```diff
@@ -1,3 +1,22 @@
+from django.contrib.auth.base_user import AbstractBaseUser
 from django.db import models
 
-# Create your models here.
+from common.models import TimeStampedModel
+
+
+class Users(TimeStampedModel, AbstractBaseUser):
```

Comment:
User

Reply:
> User

Does it really should be "User"  instead "Users" as in ER diagram?